client/my-sites/email/email-management/index.jsx

```diff
@@ -137,6 +137,17 @@ class EmailManagement extends React.Component {
 				secondaryAction: translate( 'Add Email Forwarding' ),
 				secondaryActionURL: emailManagementForwarding( selectedSiteSlug, selectedDomainName ),
 			};
+			if ( isMappedDomain( selectedDomain ) ) {
+				Object.assign( emptyContentProps, {
+					line: translate(
+						'Only domains using WordPress.com name servers are eligible for G Suite.'
+					),
+					action: translate( 'How To Change Name Servers' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **24** times:
`translate( 'How to change your name servers' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).